pid_controller/include/pid_controller/pid_controller.hpp

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2023, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Authors: Daniel Azanov, Dr. Denis
+//
+
+#ifndef PID_CONTROLLER__PID_CONTROLLER_HPP_
+#define PID_CONTROLLER__PID_CONTROLLER_HPP_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "control_msgs/msg/multi_dof_command.hpp"
+#include "control_msgs/msg/multi_dof_state_stamped.hpp"
+#include "control_toolbox/pid_ros.hpp"
+#include "controller_interface/chainable_controller_interface.hpp"
+#include "pid_controller/visibility_control.h"
```

Comment:
are we strict with google codestyle? When I got it right, the _pid_controller_ - package dependencies have to be the last ones

Reply:
we could add them in the separate section, but I tend to have alphabetic sort.